server/auth.js

```diff
@@ -25,6 +25,7 @@ const init = oauthApi => {
             clientSecret: config.apis.oauth2.clientSecret,
             callbackURL: `${config.app.url}/login/callback`,
             state: true,
+            sessionKey: 'oauth2',
```

Comment:
session key defaults to `ouath2:auth_domain`, I presume to allow auth to multiple domains simultaneously.  We therefore keep it simple so can easily lookup the session key later.